src/vm/comutilnative.cpp

```diff
@@ -1922,6 +1922,21 @@ FCIMPL0(int, GCInterface::GetMaxGeneration)
 }
 FCIMPLEND
 
+FCIMPL0(INT64, GCInterface::GetAllocatedBytesForCurrentThread)
+{
+    FCALL_CONTRACT;
+
+    INT64 currentAllocated = 0;
+    Thread *pThread = GetThread();
+    if (pThread)
```

Comment:
The null check is not necessary - the unmanaged Thread object is guaranteed to exist inside FCalls.